meta-genivi-dev/recipes-dev-platform/images/genivi-dev-platform.bb

```diff
@@ -32,6 +32,7 @@ IMAGE_INSTALL_append = " \
     packagegroup-gdp-qt5 \
     packagegroup-gdp-rvi \
     packagegroup-gdp-dev \
+    chromium-wayland \
```

Comment:
I like this order better because it guarantees that Chromium is built with the GDP specific bits that we added in the previous patches. Previous patches alone don't break, they just allow to build Chromium optionally. Anyway, it doesn't make a big difference, the build shouldn't break in any order.